applications/nrf5340_audio/overlay-headset.conf

```diff
@@ -13,13 +13,15 @@ CONFIG_LC3_DEC_CHAN_MAX=1
 
 # ***** CIS *****
 # --- REMOVE THE ONES THAT ARE NOT NEEDED ---
-# CONFIG_BT_PERIPHERAL=y
-# CONFIG_BT_VCS=y
-# CONFIG_BT_GAP_PERIPHERAL_PREF_PARAMS=n
-# CONFIG_BT_MAX_CONN=1
-# CONFIG_BT_ISO_MAX_CHAN=1
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_GAP_PERIPHERAL_PREF_PARAMS=n
+CONFIG_BT_MAX_CONN=1
+CONFIG_BT_ISO_MAX_CHAN=1
+CONFIG_BT_AUDIO_UNICAST_SERVER=y
```

Comment:
Looks like we do not need CONFIG_TRANSPORT_CIS anymore?
Or use this config to select other .conf files?

Reply:
The CMakeLists.txt under bluetooth folder still rely on this config to include CIS or BIS source file.

Reply:
Yeah, sorry, I forgot that CONFIG_TRANSPORT_CIS is default, and do not have to be set